openid_connect.js

```diff
@@ -7,7 +7,38 @@ var newSession = false; // Used by oidcAuth() and validateIdToken()
 
 export default {auth, codeExchange, validateIdToken, logout};
 
+function sleep(ms) {
+    return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+async function isVarDefined(variable) {
+    const v = await sleep(10);
```

Comment:
May be poll less often, say 100? That still will be unnoticeable for most people and observed delay will be primarily defined by zone_sync_interval anyway.

Reply:
Reasonably, changed to 100ms.